### PR TITLE
Rewrite ctcp handling

### PR DESCRIPTION
**Pull request by xPaw**
Also fixes PING not replying with all arguments.

### CODE REVIEW

File: src/plugins/irc-events/ctcp.js [hunk 1]
```diff
@@ -1,6 +1,18 @@
 "use strict";
 
+const Helper = require("../../helper");
 const Msg = require("../../models/msg");
+const pkg = require("../../../package.json");
+
+const ctcpResponses = {
+	CLIENTINFO: () => Object // TOOD: This is currently handled by irc-framework
```
Comment (dgw) [inline src/plugins/irc-events/ctcp.js]: `TOOD`?

Comment (xPaw) [reply src/plugins/irc-events/ctcp.js]: https://github.com/kiwiirc/irc-framework/issues/115

Comment (MaxLeiter) [reply src/plugins/irc-events/ctcp.js]: I think @dgw was pointing out it should be TODO

Comment (astorije) [reply src/plugins/irc-events/ctcp.js]: Fixed.

**Comment by astorije** [comment]
Rebased on `master` to fix tests.

**Pull request merged**